documentation/Development.md

```diff
@@ -2,7 +2,7 @@
 
 In this section we will describe how you can implement your own tutorials.
 Learn more about the file system and where you can add your commands in 
-https://github.com/devonfw-forge/tutorials/wiki/Structure
+https://github.com/devonfw-forge/tutorial-compiler/wiki/Structure
```

Comment:
https://github.com/devonfw-tutorials/tutorial-compiler/wiki/Structure ?

Reply:
that's the page, where we explain the file structure and 'working directory' and 'workspace directory'

Reply:
Yes, but the link is wrong. It should be devonfw-tutorials not devonfw-forge